components/Storages.js

```diff
@@ -1,19 +1,19 @@
 import { nanoid } from 'nanoid';
 import Link from 'next/link';
+import useLocalStorageState from 'use-local-storage-state';
 
-export default function Storages({ storages, setStorages }) {
+export default function Storages({ storages, onStorage }) {
```

Comment:
Maybe you could name this components StoragesPage because you named the WarningPage with "Page" and I think it's a little more descriptive and understandable :)